spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigClientProperties.java

```diff
@@ -116,6 +116,13 @@
 	 */
 	private String[] uri = { "http://localhost:8888" };
 
+	/**
+	 * The strategy to use when call to server fails and there are multiple URLs
+	 * configured on the uri property (default
+	 * {@link MultipleUriStrategy#CONNECTION_TIMEOUT_ONLY}).
+	 */
+	private MultipleUriStrategy multipleUriStrategy = MultipleUriStrategy.CONNECTION_TIMEOUT_ONLY;
```

Comment:
Since this is going into main I wonder if the strategy can be `ALWAYS` by default?  @spencergibb what do you think?

Reply:
Sounds good to me

Reply:
Will change it. (I was thinking to keep current behavior, but I like ALWAYS as the default better.)

Reply:
We can introduce breaking changes in the main branch